src/pathfora.js

```diff
@@ -2824,10 +2824,32 @@
      * @throws {Error} error
      * @param {string} accountId  Lytics account ID
      */
-    recommendContent: function (accountId, params, callback) {
+    recommendContent: function (accountId, params, id, callback) {
       // Recommendation API:
       // https://www.getlytics.com/developers/rest-api#content-recommendation
 
+      // if we have the recommendation response cached in session storage
+      // use that instead of making a new API request
+      var storedRec = sessionStorage.getItem('PathforaRecommend_' + id);
+
+      if (typeof storedRec === 'string' && params.visited !== false) {
+        var rec = JSON.parse(storedRec);
```

Comment:
probably should try/catch here to be safe. a plain string will blow up with JSON.parse if its not actually JSON. should be fine since its our api but im thinking we want this as bulletproof as possible?